### PR TITLE
Course editing

### PR DESCRIPTION
**Pull request by Nouraal21**
added a view for editing a course, a template and a url

### CODE REVIEW

File: courses/templates/course_edit.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %} {% block content %}
+<div>
+  <h1> Course Edit </h1>
+  </br>
+  <form method="post" action="{% url 'course_edit' course.pk %}">
```
Comment (almarzouq) [inline courses/templates/course_edit.html:5]: you are using the context_object_name course here
look at:
    course.pk

File: courses/views.py [hunk 2]
```diff
@@ -154,3 +155,10 @@ def student_can_add_course(request, course_id, student_id):
         student.save()
         messages.success(request, 'You are enrolled in %s' % (course))
     return redirect('/')
+
+
+class CourseEdit(UpdateView):
+    model = Course
+    template_name = "course_edit.html"
+    # context_object_name = "course"
```
Comment (almarzouq) [inline courses/views.py]: this is used in the template ... you have a variable named course ... it holds the object that will be edited

File: courses/templates/course_edit.html (hunk 1 above)
Comment (Nouraal21) [inline courses/templates/course_edit.html:5]: did you mean i need to un comment the context_object?

Comment (Nouraal21) [inline courses/templates/course_edit.html:5]: pushed new commit hope its right now

**Pull request merged**